src/api/entities/SecurityToken/types.ts

```diff
@@ -48,4 +48,9 @@ export interface TransferBreakdown {
   result: boolean;
 }
 
+export interface ExternalAgent {
```

Comment:
`AgentWithGroup`

Also, there is an `Agent` entity. You should use that instead of `Identity`